profiler/src/ProfilerEngine/Datadog.Linux.ApiWrapper/CMakeLists.txt

```diff
@@ -19,13 +19,6 @@ add_compile_options(-std=c11 -fPIC)
 SET(API_WRAPPER_BASENAME Datadog.Linux.ApiWrapper)
 SET(API_WRAPPER_SHARED_LIB_NAME ${API_WRAPPER_BASENAME}.x64)
 
-# Deployment directory
-SET(DEPLOY_DIR ${OUTPUT_BUILD_DIR}/DDProf-Deploy)
```

Comment:
Since it is used below, if you don't define DEPLOY_DIR here, where will it come from?

Reply:
`DEPLOY_DIR` is defined in the `profiler/CMakeLists.txt`.